src/bloaty.h

```diff
@@ -337,18 +337,31 @@ void DisassembleFindReferences(const DisassemblyInfo& info, RangeSink* sink);
 
 class Rollup;
 
+struct DomainSizes {
+  int64_t vm;
+  int64_t file;
+};
+
 struct RollupRow {
   RollupRow(const std::string& name_) : name(name_) {}
 
   std::string name;
-  int64_t vmsize = 0;
-  int64_t filesize = 0;
-  int64_t filtered_vmsize = 0;
-  int64_t filtered_filesize = 0;
+  DomainSizes size = {0, 0};
+  DomainSizes filtered_size = {0, 0};
+
   int64_t other_count = 0;
   int64_t sortkey;
   double vmpercent;
   double filepercent;
+
+  // The size of this row in a diff base.  Same as `size` for non-diff.
+  DomainSizes old_size = {0, 0};
```

Comment:
What if we stored only `old_size` or only `new_size`, and let the other get derived from that?

I wonder if it might help make the code a little simpler.

Reply:
In particular, you wouldn't need to fill in either for a non-diff run.

Reply:
Storing only one struct and deriving the other by adding the `size` would be done during printing as we will not have a struct to hold those values. I am not sure, but should we refrain from doing any kind of logic or calculations within a printing function? Or is there another way of obtaining those sizes?